integration/integration_test.go

```diff
@@ -312,7 +340,7 @@ func (s *BaseSuite) createRequiredNamespaces(c *check.C) {
 	s.kubectlCommand(c, "create", "namespace", testNamespace)
 }
 
-func (s *BaseSuite) installHelmMaesh(c *check.C, smi bool, kubeDNS bool) error {
+func (s *BaseSuite) installHelmMaesh(c *check.C, smi bool, kubeDNS bool, acl bool) error {
```

Comment:
```suggestion
func (s *BaseSuite) installHelmMaesh(c *check.C, kubeDNS bool, acl bool) error {
```

Reply:
Actually, I think we should keep this parameter and add a new test which checks if the `--smi` option is still handled correctly.

Reply:
Yes, I'd prefer that

Reply:
Done